dts/arm/st/stm32l432.dtsi

```diff
@@ -5,3 +5,20 @@
  */
 
 #include <st/stm32l4.dtsi>
+#include <dt-bindings/clock/stm32_clock.h>
+
+/ {
+	soc {
+		can1: can@40006400 {
```

Comment:
If I'm correct, all STM32L4 SoC have this CAN instance and hence it could be put in stm32l4xx.dtsi

Reply:
STM32L462CE does not have any CAN.
Should I move it anyway?

Reply:
ok, had miss this one and STM32L462RE
But, if I'm correct, this will be the only 2 without CAN support in L4 series.
Rather than adding it all other files, we can use `/delete-node/ &can_1;` in stm32l462re/ce.dtsi files.

Reply:
As far as I can see on this page all of the L4 series have CAN
http://www.st.com/en/microcontrollers/stm32l4-series.html?querycriteria=productId=SS1580

In particular both STM32L462CE and STM32L462RE have it listed on their page as well.

Reply:
@karstenkoenig I think you are right!

Reply:
Well, indeed, CAN is not mentioned as supported on the table while it is on the respective pages.
So anyway, let's move can node in stm32l4.dtsi and if required they can be deleted in stm32l462(c/r)e.dtsi files.
I'll try to get someone clarifying information on the website.

Reply:
@alexanderwachter , ok, thanks